frontend/src/component/Sidebar/SidebarSpaceItem.jsx

```diff
@@ -125,13 +125,13 @@ class SidebarSpaceItem extends React.Component {
   render () {
     const { props, state } = this
     const INDENT_WIDTH = 20
-    const BASE_MARGIN = 20
+    const BASE_MARGIN = 25
 
     return (
-      <div
+      <Link
         id={props.id}
         className={classnames(
-          'sidebar__item__space',
+          'sidebar__item__inside_menu sidebar__item__space',
```

Comment:
_Not a request change_
I feel like we starting to cumulate classes while we could have a specific selector.
For example: ``sidebar__title sidebar__item {your css}`` (or something like that, I didn't tested the code)

Reply:
I'm not saying that classes can't be simplified, but in principle they all have a reason to exist (they regroup behaviors in common). Instead of recopying all the common features of a sidebar item when it is a title or when it is inside a space, what is done is that we have the item class with the common features and the title and inside_menu classes for the specific features. So when we change the font size of all the items for example, we can change it in one place.